src/composables/QuasarFieldRule.js

```diff
@@ -20,3 +20,14 @@ export function isGitRepositoryUrl(t, value) {
   return /^http(s)?:\/\/.+\/.*(?<!\.git)$/.test(value)
     || t('errors.invalid.gitProvider.repository');
 }
+/**
+ * Check if branch name is not already exists.
+ * @param {Function} t - I18n translate function.
+ * @param {String} value - Value to check.
+ * @return {boolean|String} Return true if the branch not already exists otherwise the translated
```

Comment:
```suggestion
 * @return {boolean|String} Return true if the branch does not already exist otherwise the translated
```

Reply:
I change the documentation and the Function name